estate/models/estate_property.py

```diff
@@ -0,0 +1,43 @@
+from odoo import fields, models
+
+class EstateProperty(models.Model):
+    _name = "estate_property"
+    _description = "Real estate management"
+
+    active = True
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(
+        copy=False,
+        default=fields.Date.add(fields.Date.today(), months=3)
+    )
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        string='Orientation',
+        selection=[
+            ('north', 'North'),
+            ('east', 'East'),
+            ('south', 'South'),
+            ('west', 'West')
+        ]
+    )
+    state = fields.Selection(
+        selection =[
+            ('new', 'New'),
+            ('offer_received', 'Offer Received'),
+            ('offer_accepted', 'Offer Accepted'),
+            ('sold', 'Sold'),
+            ('cancelled', 'Cancelled'),
+        ],
+        copy=False,
+        default='new',
+        required=True
+    )
```

Comment:
```suggestion
    state = fields.Selection([
        ('new', 'New'),
        ('offer_received', 'Offer Received'),
        ('offer_accepted', 'Offer Accepted'),
        ('sold', 'Sold'),
        ('cancelled', 'Cancelled'),
    ], copy=False, default='new', required=True, string="Status")
```
just a matter of coding style. to gain some lines without being unreadable :)